packages/plugin-hardhat/src/defender/get-approval-process.ts

```diff
@@ -0,0 +1,52 @@
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+
+import { getNetwork, getDeployClient } from './utils';
+import { ApprovalProcessResponse } from '@openzeppelin/defender-sdk-deploy-client';
+
+export interface ApprovalProcess {
+  approvalProcessId: string;
+  address?: string;
```

Comment:
I think it would be useful for end users to include `viaType` too

viaType: "EOA" | "Multisig" | "Safe" | "Relayer"